pramen/core/src/main/scala/za/co/absa/pramen/core/pipeline/PipelineDef.scala

```diff
@@ -31,6 +31,7 @@ object PipelineDef {
   val ENVIRONMENT_NAME = "pramen.environment.name"
   val PIPELINE_NAME_KEY = "pramen.pipeline.name"
   val OPERATIONS_KEY = "pramen.operations"
+  val TENANT_KEY = "pramen.tenant"
```

Comment:
You added a key for tenant, but not a field to this class (`PipelineDef`). Let's also add the field.